RecoLocalCalo/HcalRecAlgos/src/HcalMahiPulseOffsetsGPU.cc

```diff
@@ -0,0 +1,36 @@
+#include "RecoLocalCalo/HcalRecAlgos/interface/HcalMahiPulseOffsetsGPU.h"
+
+#include "FWCore/Utilities/interface/typelookup.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/cudaCheck.h"
+
+// FIXME: add proper getters to conditions
+HcalMahiPulseOffsetsGPU::HcalMahiPulseOffsetsGPU(edm::ParameterSet const& ps) 
+{
+    auto const& values = ps.getParameter<std::vector<int>>("pulseOffsets");
+    values_.resize(values.size());
+    std::copy(values.begin(), values.end(), values_.begin());
+}
```

Comment:
```suggestion
HcalMahiPulseOffsetsGPU::HcalMahiPulseOffsetsGPU(edm::ParameterSet const& ps) :
    values_(ps.getParameter<std::vector<int>>("pulseOffsets"))
{
}
```

But I do agree with @makortel , it would be better to pass directly the `std::vector<int>` rather than the `edm::ParameterSet`.